ruby_on_rails/rails_sprinkles/js_bundling.md

```diff
@@ -14,29 +16,46 @@ This section contains a general overview of topics that you will learn in this l
 
 ### JS Bundling Options
 
-Before we install the jsbundling-rails gem let's go over a few of the options. <span id='bundle-options'></span>The three options are esbuild, rollup and webpack. Esbuild is a fast lightweight bundler written in the GO language whereas other bundlers may be slower, but have more features which can make the bundler take longer to run. Esbuild manages to be fast through being built from scratch and a variety of methods such as utilizing parallelism to use all available CPU cores. If you're interested in reading more about esbuild you can find the documentation [here](https://esbuild.github.io/). Rollup is another bundler that allows you to utilize a JavaScript syntax for importing and exporting functions and data so they can be shared between separate scripts then changing them to existing supported formats. If you want to take a deeper dive into how to utilize Rollup you can find the documentation [here](https://rollupjs.org/guide/en/#introduction). The final bundler provided through jsbundling-rails is webpack which is a static module bundler that uses an entry point within your application to create a dependency graph that then combines every module your project needs into one or more bundles to be used by your application. If you want to take a deeper dive into webpack go [here](https://webpack.js.org/concepts/). Now that you have been introduced into the bundlers provided through jsbundling-rails let's go into setting it up.
+Before we install the jsbundling-rails gem let's go over a few of the options. <span id='bundle-options'></span>The three options are:
+
+#### Esbuild
+Esbuild is a fast lightweight bundler written in [Go](). Whereas other bundlers may be slower, but have more features which can make the bundler take longer to run. Esbuild manages to be fast with variety of methods such as utilizing parallelism to use all available CPU cores.
```

Comment:
Looks like you left a spot for a link. Not sure if you were wanting a different link, or maybe its documentation?
https://go.dev/doc/

Reply:
Get spot, thanks Rachel 💪